docs/_content/ontwikkelaars/aan-de-slag.md

```diff
@@ -18,7 +18,7 @@ start te gaan. Ga anders naar de **Voorbereiding**.
 $ git clone git@github.com:VNG-Realisatie/gemma-zaken.git
 $ cd gemma-zaken/infra
 $ docker-compose pull
-$ docker-compose up -d
```

Comment:
@joeribekker deze heb ik weggehaald omdat ze anders alle logging/error output verbergen en het niet duidelijk is of een service nu wel of niet draait